app/views/groups/_form.html.erb

```diff
@@ -1,9 +1,15 @@
 <%= form_for(@group) do |f| %>
-<% if @group.errors.any? %>
-  <div class="error_explanation">
-    <%= t('.error_explanation') %>
-  </div>
-<% end %>
+  <% if @group.errors.any? %>
+    <div class="error_explanation">
+      <h2><%= pluralize(@group.errors.count, 'error') %>
+        prohibited this group from being saved:</h2>
```

Comment:
I copied this from RailsGuids, so if we want different phrasing, I can update it

http://guides.rubyonrails.org/active_record_validations.html#displaying-validation-errors-in-views